packages/larva-patterns/modules/article-tags/article-tags.twig

```diff
@@ -1,3 +1,3 @@
 <div class="article-tags // {{ article_tags_classes }}">
 	{% include "@larva/objects/o-nav/o-nav.twig" with o_nav %}
-</div>
\ No newline at end of file
+</div>
```

Comment:
Are these new line diffs adding or removing ending empty lines? We want to make sure these patterns do have ending empty lines for when then they are run through PHPCS

Reply:
Hey @laras126 

Here we are adding an empty line at end of the file to resolve PHPCS issues.